src/main/java/com/_up/megastore/services/interfaces/IUserService.java

```diff
@@ -13,5 +13,5 @@ public interface IUserService {
   void activateUser(UUID userId, UUID activationToken);
 
   void sendEmailToRecoverPassword(String email);
-  void recoverPassword(UUID userId, String newPassword, UUID recoverPasswordToken);
+  void recoverPassword(UUID userId, String password, String confirmPassword, UUID recoverPasswordToken);
```

Comment:
Ya que tenemos el RecoverPasswordRequest podríamos recibir el objeto completo en vez de sus atributos por separado
```suggestion
  void recoverPassword(UUID userId, RecoverPasswordRequest recoverPasswordRequest);
```